Rakefile

```diff
@@ -68,6 +68,8 @@ task :docs do
                                      'bolt-modules/system'])
   json = JSON.parse(File.read(tmpfile))
   funcs = json.delete('puppet_functions')
+  # TODO: With puppet-strings >= 2.3.0 data type docs are generated - do we want/need to do anything with this?
```

Comment:
Seems reasonable for now to me.

Reply:
This is fine for now. We do have reference docs for the datatypes I think we should ticket switching those out for the strings docs separately.